### PR TITLE
docs: updates to Banner, ContentContainer, LayoutGrid, RadialProgress, SkipToContent, Text

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Adds Storybook docs per pairings with Reach friends.

### PR Checklist

- ~[ ] Related to designs:~
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@23.1.2-alpha.d3e9a1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601961c5d5e340aa17d3fb05--gamut-preview.netlify.app>https://601961c5d5e340aa17d3fb05--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601961c5d5e340aa17d3fb05>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@23.1.2-alpha.c85c65.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601b4f31f46e14626161cdd7--gamut-preview.netlify.app>https://601b4f31f46e14626161cdd7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601b4f31f46e14626161cdd7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@23.1.2-alpha.62733e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601c665d19f9d09842b6cd00--gamut-preview.netlify.app>https://601c665d19f9d09842b6cd00--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601c665d19f9d09842b6cd00>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601d4a3f2e6289fe009b4471--gamut-preview.netlify.app>https://601d4a3f2e6289fe009b4471--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601d4a3f2e6289fe009b4471>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@25.1.1-alpha.ebf345.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@25.1.1-alpha.e3fd23.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://601d58ab095b3202849e3812--gamut-preview.netlify.app>https://601d58ab095b3202849e3812--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/601d58ab095b3202849e3812>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
these are all great improvements + address the 'whys' + 'hows' of these components really clearly. super!

**Review by codecaaron — APPROVED** [review]

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60254250a92bf928d47adc57--gamut-preview.netlify.app>https://60254250a92bf928d47adc57--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60254250a92bf928d47adc57>Deploy Logs</a></p>

File: packages/styleguide/stories/Layouts/ContentContainer.stories.mdx [hunk 1]
```diff
@@ -13,10 +13,13 @@ import { Canvas, Meta, Props, Story } from '@storybook/addon-docs/dist/blocks';
   }}
 />
 
-ContentContainer can used when creating page layouts to contain content
-within a maximum width, as well as centering that content and providing
+ContentContainer can be used when creating page layouts to contain content
+within the \$grid-xl maximum width of [1440px](/?path=/docs/foundations-layout--page#screensizes-and-media-queries), as well as centering that content and providing
```
Comment (rogervera) [inline packages/styleguide/stories/Layouts/ContentContainer.stories.mdx:17]: Looks a bit weird here. Was this supposed to be?

```suggestion
within the `$grid-xl` maximum width of [1440px](/?path=/docs/foundations-layout--page#screensizes-and-media-queries), as well as centering that content and providing
```
Screen Shot 2021-02-11 at 9 45 05 AM

Ex: https://601d58ab095b3202849e3812--gamut-preview.netlify.app/storybook/?path=/docs/layouts-contentcontainer--content-container

**Review by rogervera — APPROVED** [review]
:shipit:

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@25.1.1-alpha.1aa5ae.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60254595632bfe26381b0d76--gamut-preview.netlify.app>https://60254595632bfe26381b0d76--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60254595632bfe26381b0d76>Deploy Logs</a></p>

**Review by aresnik11 — APPROVED** [review]
Looks great, love the changes and additions!

**Review by christian-dinh — APPROVED** [review]
📚

**Review by judahtanthony — APPROVED** [review]
Thanks for driving this improvements.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@25.2.2-alpha.952ebc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60255c6c7101d73bf4d8ddcf--gamut-preview.netlify.app>https://60255c6c7101d73bf4d8ddcf--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60255c6c7101d73bf4d8ddcf>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@26.1.1-alpha.6e9748.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://602c0e17b73b6c64552c272a--gamut-preview.netlify.app>https://602c0e17b73b6c64552c272a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/602c0e17b73b6c64552c272a>Deploy Logs</a></p>

**Pull request merged**